services/api-server/openapi.json

```diff
@@ -4397,6 +4509,26 @@
         "type": "object",
         "title": "Links"
       },
+      "LogLinkMap": {
+        "properties": {
+          "map": {
+            "additionalProperties": {
```

Comment:
The problem of using a `dict` (or a Map) as you call it, is that the OAS look very confusing since it defines only a generic `object`.

I would rather go for a concrete object that represent a `LogLink` e.g.
```python
class LogLink(BaseModel):
     node_name: NodeName
     donwload_link: DownloadLink
     
class JobLogsMap(BaseModel):
     job_name: URL # I know this is redundant since it is in the path but it is useful to have it in the result
     log_links: dict[NodeName, LogLink] | None = None
```

This will render much clearer in the OAS!

Reply:
I have changed this to 
```python
class LogLink(BaseModel):
    node_name: NodeName
    download_link: DownloadLink


class JobLogsMap(BaseModel):
    log_links: list[LogLink] = Field(..., description="Array of download links")
```
to make the models more clear from the OAS